README.md

```diff
@@ -22,24 +22,24 @@ yarn add @chainsafe/bls @chainsafe/blst
 By default, native bindings will be used if in NodeJS and they are installed. A WASM implementation ("herumi") is used as a fallback in case any error occurs.
 
 ```ts
-import {SecretKey, secretKeyToPublicKey, sign, verify} from "@chainsafe/bls";
+import bls, {init} from "@chainsafe/bls/switchable"
 
 (async () => {
-  await init("herumi");
+    await init("herumi");
```

Comment:
Can you use the default export and remove the `init`? I think this example should show the simplest possible usage (which handles the initialization transparently)